src/components/Header/UserActionList/index.tsx

```diff
@@ -14,11 +15,16 @@ type UserActionProps = {
 };
 
 export const UserActionList = ({
+  id,
   className,
   children
 }: UserActionListProps) => {
   const classNames = classnames('user-actions-dropdown', className);
-  return <ul className={classNames}>{children}</ul>;
+  return (
+    <ul id={id} className={classNames}>
```

Comment:
Adding `id` so there is a relationship between the `button` and the element it is opening.